src/components/network/equipment-table-editors.js

```diff
@@ -82,12 +90,76 @@ export const NumericalField = ({
                     style: {
                         textAlign: 'center',
                         fontSize: 'small',
-                        flexGrow: 1,
                     },
                     min: { min },
                     max: { max },
+                    step: 'any',
                 }}
             />
-        </Tooltip>
+        );
+    }
+
+    return (
+        <div>
+            {min !== undefined || max !== undefined ? (
```

Comment:
If there is no "min" values, there should not be "min: " in the tooltip.

Reply:
agreed. to be changed

Reply:
Maybe change the new code to not have a lot of ternary operators ?

Reply:
agreed. to be changed in next commit